pkg/controller/deadmanssnitchintegration/deadmanssnitchintegration_controller.go

```diff
@@ -123,6 +126,13 @@ type ReconcileDeadmansSnitchIntegration struct {
 func (r *ReconcileDeadmansSnitchIntegration) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling DeadmansSnitchIntegration")
+
+	if len(os.Getenv("FEDRAMP")) > 0 {
```

Comment:
Should check for zero length here:
```suggestion
	if len(os.Getenv("FEDRAMP")) == 0 {
```